tla/consistency/TraceMultiNodeReads.cfg

```diff
@@ -0,0 +1,48 @@
+\* A trace specification defines a set of one or more traces, wherein the value of some
+\* variables in each state aligns with the values specified on the corresponding line
+\* of the log or trace file. Commonly, the log file might be incomplete, failing to
+\* provide the value of all variables. In such instances, the omitted values are
+\* determined non-deterministically, adhering to the high-level specification's parameters.
+\* Furthermore, a trace may not be complete such that it only matches a prefix of the log.
+SPECIFICATION
+    TraceSpec
+
+POSTCONDITION
+    TraceMatchedNonTrivially
+
+\* Some of the traces in TraceSpec may not refine the high-level specification ccfraft.
```

Comment:
I think these comments are copy-pasted, because they reference `ccfraft`. Can they be generalised or removed?

Reply:
Oops yes.

Reply:
@eddyashton because I am slightly curious, I am going to:

1. not fix this
2. merge this PR
3. see if #6132 catches it
4. fix it afterwards, regardless of the outcome

Reply:
I don't think #6132 will catch it, because it only lints changed lines.

Reply:
#6135